python/redmonster/datamgr/io2.py

```diff
@@ -1221,6 +1254,11 @@ def merge_chi2(self):
             topdir = None
             print("'REDMONSTER_SPECTRO_REDUX' env variable not set.")
         try:
+            rmver = environ['REDMONSTER_VER']
```

Comment:
Hmm, we don't currently set `REDMONSTER_VER` or `REDMONSTER_SPECTRO_REDUX` in our Module files.  This wouldn't be hard to do, but it raises the question of whether there are other environment variables we are missing.